README.md

```diff
@@ -1 +1,6 @@
-# Assignment-Design-Pattern
\ No newline at end of file
+# Assignment-Design-Pattern
+
+Design Pattern used in program are listed below:
+1. Factory Pattern
+2. Decorator Pattern
+3. ObserverPattern
```

Comment:
Right design pattern, however should have been named better (both class and methods).